app/locale/en.coffee

```diff
@@ -1527,6 +1527,8 @@
   sharing:
     game: "Game"
     webpage: "Webpage"
+    your_students_preview: "Your students click here to preview their creation as their classmates would see it! Unavailable in teacher preview."
```

Comment:
`Your students will click here to see their finished projects! Unavailable in teacher preview.`